api/src/main/java/org/openmrs/module/ugandaemrreports/library/HIVCohortDefinitionLibrary.java

```diff
@@ -600,9 +600,9 @@ public CohortDefinition getPatientsWhoHadAViralLoadTestPeriodAfterArtInitiationA
         return cd;
     }
 
-    public CohortDefinition getPatientsWhoHadAViralLoadTestDuringThePastPeriodFromEndDate(String pastPeriods) {
-        return df.getPatientsWhoseObsValueDateIsBetweenPastPeriodFromEndDate(hivMetadata.getViralLoadDate(), hivMetadata.getARTEncounterPageEncounterType(), pastPeriods, BaseObsCohortDefinition.TimeModifier.ANY);
-    }
+   public CohortDefinition getPatientsWhoHadAViralLoadTestDuringThePastPeriodFromEndDate(String pastPeriods){
+       return df.getPatientsWhoseObsValueDateIsBetweenPastPeriodFromEndDate(hivMetadata.getViralLoadDate(),hivMetadata.getARTEncounterPageEncounterType(),pastPeriods, BaseObsCohortDefinition.TimeModifier.ANY);
+   }
 
     public CohortDefinition getPatientsOnDTGRegimenInPeriod() {
         return df.getPatientsWithCodedObsDuringPeriod(hivMetadata.getCurrentRegimen(), hivMetadata.getARTEncounterPageEncounterType(), Dictionary.getConceptList(Metadata.Concept.DTG_TLD_REGIMEN_LIST), BaseObsCohortDefinition.TimeModifier.ANY);
```

Comment:
@ssmusoke also this Time Modifier, shouldnt it be Last instead of Any

Reply:
@solemabrothers We are using any since once you take DTG in the period, you have been transitioned. If you use LAST it may cause issues if the patient has moved to another regimen